client/src/pages/about/index.js

```diff
@@ -74,30 +72,67 @@ export const About = () => {
         width={{ width: 'full', max: 'xlarge' }}
       >
         <Box pad={responsive({ horizontal: 'medium' })}>
-          <Text weight="bold">
-            ALSF funds cutting-edge pediatric cancer research
-          </Text>
-          <Text>
-            Alex’s Lemonade Stand Foundation (ALSF) funded 10 childhood cancer
-            investigators from eight different institutions working on
-            single-cell profiling to create a publicly available atlas of
-            single-cell pediatric cancer data.{' '}
-            <Link
-              href={config.links.grants_program}
-              label="Learn more about the grants
+          <Box margin={{ bottom: responsive('xlarge', '76px') }}>
+            <Text weight="bold">
+              ALSF funds cutting-edge pediatric cancer research
+            </Text>
+            <Text>
+              In 2019, Alex’s Lemonade Stand Foundation (ALSF) funded 10 grants
+              for investigators working on single-cell profiling of patient
+              samples to create a publicly available atlas of single-cell
+              pediatric cancer data. .
+            </Text>
+            <Text margin={{ top: 'medium' }}>
+              <Link
+                href={config.links.grants_program}
+                label="Learn more about the grants
             program"
-            />
-            .
-          </Text>
+              />
+            </Text>
+          </Box>
+          <Box margin={{ bottom: responsive('xlarge', '76px') }}>
+            <Text weight="bold">The Data Lab processes the generated data</Text>
+            <Text>
+              In 2019, Alex’s Lemonade Stand Foundation (ALSF) funded 10 grants
+              for investigators working on single-cell profiling of patient
+              samples to create a publicly available atlas of single-cell
+              pediatric cancer data.
+            </Text>
+            <Text margin={{ top: 'medium' }}>
+              <Link
+                href={config.links.scpca}
+                label="View the current number of tumor types represented and samples being processed here."
```

Comment:
```suggestion
                label="View the current number of tumor types represented and samples being processed here"
```

Getting rid of the full stop on the link